model/src/index.ts

```diff
@@ -71,6 +72,16 @@ export const platforma = BlockModel.create()
     },
   )
 
+  .retentiveOutput(
+    'availableColumns',
+    (ctx) => {
+      return Object.fromEntries(ctx.prerun
```

Comment:
What if argument of `fromEntries` is undefined?

Reply:
Added additional optional handling